runtime/services/catalog/migrator/migrator.go

```diff
@@ -80,7 +81,12 @@ func GetDependencies(ctx context.Context, olap drivers.OLAPStore, catalog *drive
 		// no error here. not all migrators are needed
 		return []string{}
 	}
-	return migrator.GetDependencies(ctx, olap, catalog)
+	// convert dependencies to lower case here
+	dependencies := migrator.GetDependencies(ctx, olap, catalog)
+	for i, dep := range dependencies {
+		dependencies[i] = strings.ToLower(dep)
+	}
+	return dependencies
```

Comment:
Maybe safer to do the lowercase/normalization in the migration logic and then continue returning the outright values here? To make it easier to reason about when names are normalized and when they are not